spec.bs

```diff
@@ -1436,6 +1451,25 @@ trust that the merchant showed the user the correct amount in their checkout
 flow (and any fraud discoveries are post-payment, when the user checks their
 account statement).
 
+## Lack of user activation requirement ## {#sctn-security-user-activation-requirement}
+
+If the user agent decides to skip the user activation requirement as outlined in
+[[#sctn-modify-user-activation-requirement]], some additional security mitigations
+should be considered. Removing this requirement increases the risk of spam and
+click-jacking attacks, by allowing a Secure Payment Confirmation flow to be
+initiated without the user's intent.
```

Comment:
```suggestion
initiated without the user interacting with the page immediately beforehand.
```

User activation does not carry intent :). (That's basically why we're ok removing the requirement!)